### PR TITLE
WIP: Aos6 Fix Fan and Aos6-7 thresholds

### PR DESCRIPTION
**Pull request by joseUPV**
This Pull Request fixes:
- Discovery of ghost fans in aos6 with nostatus.
- Some false positive with aos6 sensors. Changed sensor thresholds.
- To do: Some false postive with aos7 sensors. Changed sensor thresholds.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Changes look good, just needs some test data to make sure they don't get broken in the future.

**Comment by joseUPV** [comment]
We have to add a modified aos7 yaml file with new thresholds to avoid some false positive alarms.

File: includes/definitions/discovery/aos6.yaml [hunk 1]
```diff
@@ -25,52 +25,68 @@ modules:
...
+                  low_limit: ddmTempLowAlarm
+                  low_warn_limit: ddmTempLowWarning
+                  high_limit: ddmTempHiAlarm
+                  warn_limit: ddmTempHiWarning
 
         voltage:
             data:
                 -
-                  oid: ddmSupplyVoltage
+                  oid: ddmInfoEntry
```
Comment (murrant) [inline includes/definitions/discovery/aos6.yaml:36]: I prefer to walk ddmInfoTable instead of "Entry" (same below)

**Comment by joseUPV** [comment]
I was waiting to have statistical data of fan speeds,temperature and other sensor to define limits, but it's taking a lot of time.
I will split this Pull Request in two parts, one to fix aos6 ghost fan, and other to fix limits in yaml files.

**Pull request closed without merging**